test/DAOracle/helpers.ts

```diff
@@ -40,48 +40,48 @@ export type Proposal = {
 export async function getProposal(
   options: Partial<Proposal> = {}
 ): Promise<Proposal> {
-  let { feedId, timestamp, value, data } = options;
+  let { indexId, timestamp, value, data } = options;
   return {
-    feedId: feedId ?? FEED_ID,
+    indexId: indexId ?? INDEX_ID,
     timestamp: timestamp ?? (await lastBlock()).timestamp,
     value: value ?? Math.round(Math.random() * 250),
-    data: data ?? hre.ethers.utils.formatBytes32String(`id:MF4-14D-ETH`),
```

Comment:
@italianst4 This change might be ok, but looking at `FEED_ID` and `INDEX_ID` diffs above, and then looking at this change here something doesn't quite line up. I'm only scanning this PR but this is the only instance of `MF4-14D-ETH` that's being changed which seems a little odd.

Reply:
@rforte I saw this and it appears to be a riff of roman numerals (4 => IV).  Given the ids are generated off-chain, (as the example here in our test cases) I think we should keep things consistent with how our id formats are structured.  I hope this makes sense.